guide/english/certifications/javascript-algorithms-and-data-structures/debugging/use-the-javascript-console-to-check-the-value-of-a-variable/index.md

```diff
@@ -3,8 +3,20 @@ title: Use the JavaScript Console to Check the Value of a Variable
 ---
 ## Use the JavaScript Console to Check the Value of a Variable
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/certifications/javascript-algorithms-and-data-structures/debugging/use-the-javascript-console-to-check-the-value-of-a-variable/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+In order to open up the Javascript Console of your Browser (chrome,firefox,safari,opera,edge) press: [here](https://github.com/freeCodeCamp/guide/blob/master/src/pages/certifications/javascript-algorithms-and-data-structures/debugging/understanding-the-differences-between-the-freecodecamp-and-browser-console/index.md)
```

Comment:
First, there are missing spaces in the browser list. More importantly, the link is pointing to the wrong URL. Even more importantly, it's not pointing to anything helpful.